packages/tezos-lwt-result-stdlib/tezos-lwt-result-stdlib.8.0~rc2/opam

```diff
@@ -0,0 +1,27 @@
+opam-version: "2.0"
+maintainer: "contact@tezos.com"
+authors: [ "Tezos devteam" ]
+homepage: "https://www.tezos.com/"
+bug-reports: "https://gitlab.com/tezos/tezos/issues"
+dev-repo: "git+https://gitlab.com/tezos/tezos.git"
+license: "MIT"
+depends: [
+  "dune" { >= "2.0" }
```

Comment:
All these packages need a dependency on `ocaml` as well. In this particular library, that needs to be <4.12.0 due to this:

```
#=== ERROR while compiling tezos-lwt-result-stdlib.8.0~rc2 ====================#
# context              2.0.7 | linux/x86_64 | ocaml-variants.4.12.0+trunk | file:///src
# path                 ~/.opam/4.12/.opam-switch/build/tezos-lwt-result-stdlib.8.0~rc2
# command              ~/.opam/4.12/bin/dune build -p tezos-lwt-result-stdlib -j 47
# exit-code            1
# env-file             ~/.opam/log/tezos-lwt-result-stdlib-1-531662.env
# output-file          ~/.opam/log/tezos-lwt-result-stdlib-1-531662.out
### output ###
#       ocamlc src/lib_lwt_result_stdlib/sigs/.sigs.objs/byte/sigs__Hashtbl.{cmi,cmo,cmt} (exit 2)
# (cd _build/default && /home/opam/.opam/4.12/bin/ocamlc.opt -w -40 -g -bin-annot -I src/lib_lwt_result_stdlib/sigs/.sigs.objs/byte -I /home/opam/.opam/4.12/lib/bytes -I /home/opam/.opam/4.12/lib/lwt -I /home/opam/.opam/4.12/lib/result -I /home/opam/.opam/4.12/lib/seq -no-alias-deps -open Sigs -o src/lib_lwt_result_stdlib/sigs/.sigs.objs/byte/sigs__Hashtbl.cmo -c -impl src/lib_lwt_result_stdlib/sigs/hashtbl.ml)
# File "src/lib_lwt_result_stdlib/sigs/hashtbl.ml", line 36, characters 18-75:
# 36 |   module Legacy : Stdlib.Hashtbl.S with type key = key and type 'a t = 'a t
#                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
# Error: In this `with' constraint, the new definition of t
#        does not match its original definition in the constrained signature:
#        Type declarations do not match:
#          type 'a t = 'a t
#        is not included in
#          type !'a t
#        Their variances do not agree.
#        File "hashtbl.mli", line 335, characters 4-14: Expected declaration
#        File "src/lib_lwt_result_stdlib/sigs/hashtbl.ml", line 36, characters 59-75:
#          Actual declaration
```